### ISSUE TITLE
Broken build on node v4

### ISSUE THREAD

**Issue by amertak** [pradel/react-responsive-modal#1]
Hi, thanks for great library.

However, latest commits broke build on node v4.x.x
Could you please also include this build into the travis?

Error:
```
npm ERR! node v4.5.0
npm ERR! npm  v2.15.9

npm ERR! peerinvalid The package jss@5.4.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer jss-camel-case@2.0.0 wants jss@^5.2.0
npm ERR! peerinvalid Peer jss-px@1.0.0 wants jss@>=3.0.0 < 4.0.0
npm ERR! peerinvalid Peer jss-vendor-prefixer@3.0.0 wants jss@^5.2.0
npm ERR! peerinvalid Peer react-jss@3.0.1 wants jss@^5.2.0
```

Thanks!

David

**Comment by pradel** [comment]
Should be fixed in v1.2.3 could you test please ?

**Comment by amertak** [comment]
Yes. It is fixed. Thank you :)
